### PR TITLE
[bitnami/moodle] Update moodle title

### PR DESCRIPTION
**Pull request by beltran-rubo**
**Description of the change**

Update Moodle title

**Checklist** 
- [not needed] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/moodle/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Moodle
+# Moodle(TM) LMS
```
Comment (jotadrilo) [inline bitnami/moodle/README.md:1]: maybe we can use the `<sup>` syntax, instead:

```
# Moodle<sup>TM</sup> LMS
```

# Moodle<sup>TM</sup> LMS

Comment (jotadrilo) [reply bitnami/moodle/README.md:1]: I am not sure if `sup` is supported in all Markdown parsers, though (it works in GitHub and GitLab)

Comment (beltran-rubo) [reply bitnami/moodle/README.md]: Thanks! done

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Pull request merged**